.github/workflows/pr_bot.yml

```diff
@@ -132,7 +134,7 @@ jobs:
     name: Destroy PR
     uses: ./.github/workflows/devcontainer_make_command.yml
     with:
-      suffix_override: ${{ needs.pr_comment.outputs.prRefId }}
+      suffix_override: flwr-${{ needs.pr_comment.outputs.prRefId }}
```

Comment:
So it's a bit more obvious when looking through the resource groups what pertains to flower - previously it would just be `rg-37952873`, `rg-683055d` etc.